searchcore/src/vespa/searchcore/proton/reference/pending_gid_to_lid_change.h

```diff
@@ -0,0 +1,44 @@
+// Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+
+#pragma once
+
+#include <vespa/searchlib/common/idestructorcallback.h>
+#include <vespa/document/base/globalid.h>
+#include <vespa/searchlib/common/serialnum.h>
+
+namespace proton {
+
+/*
+ * Class for a gid to lid change awaiting a force commit.
+ */
+class PendingGidToLidChange
+{
+    using Context = std::shared_ptr<search::IDestructorCallback>;
+    using GlobalId = document::GlobalId;
+    using SerialNum = search::SerialNum;
+
+    Context   _context;
+    GlobalId  _gid;
+    uint32_t  _lid;
+    SerialNum _serial_num;
```

Comment:
Move _serial_num up before the gid to avoid unused bytes due to alignment and smaller object size.

Reply:
Fixed.